modules/kwin.nix

```diff
@@ -0,0 +1,86 @@
+{ config, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.plasma;
+  validTitlebarButtons = {
+    longNames = [
+      "more-window-actions"
+      "application-menu"
+      "on-all-desktops"
+      "minimize"
+      "maximize"
+      "close"
+      "help"
+      "shade"
+      "keep-below-windows"
+      "keep-above-windows"
+    ];
+    shortNames = [
+      "M"
+      "N"
+      "S"
+      "I"
+      "A"
+      "X"
+      "H"
+      "L"
+      "B"
+      "F"
+    ];
```

Comment:
Here it might have been better using a set to get translations between the long and short names. If it works as it should though it probably is fine for now as we would then need to change other parts of the code though. I will see if there is a quick fix.

Reply:
Main reason I used a list is cause I already needed a list of long names for the option types so that way I don't repeat stuff, although I would be open to using sets as you said

Reply:
Ah OK. I missed that. Then this is a good solution I reckon.